docs/chainlink-keepers/best-practices.md

```diff
@@ -8,81 +8,86 @@ whatsnext:
     'Network Overview': '/docs/chainlink-keepers/overview/',
   }
 ---
-Chainlink Keepers are a powerful tool for smart contract automation. These are some useful patterns and best practices that you can employ within your Keeper-compatible smart contract.
+Chainlink Keepers are a powerful tool for smart contract automation. This guide demonstrates some useful patterns and best practices that you can employ within Keeper-compatible smart contracts.
 
-It is important to understand the gas limits for on-chain execution and off-chain computation when developing your Keeper-compatible smart contract - you can review that in [How Chainlink Keepers Work](../overview/).
+It is important to understand the gas limits for on-chain execution and off-chain computation when developing your Keeper-compatible smart contract. To learn more about the economics of Chainlink Keepers, read [How Chainlink Keepers Work](../overview/).
```

Comment:
The "How Chainlink Keepers Work" page is actually the "Network Overview" page. That strikes me as potentially confusing.

If there another way we could phrase this, or "Network Overview" to fit a little better?

Reply:
Updated in latest push.